### ISSUE TITLE
Shorthands are being reported as invalid

### ISSUE THREAD

**Issue by Th3S4mur41** [Jordan-Hall/stylelint-use-logical-spec#39]
With the upgrade from 4.1.0 to [5.0.0](https://github.com/Jordan-Hall/stylelint-use-logical-spec/releases/tag/5.0.0), global shorthands (padding, margin) are now being reported as errors

![image](https://user-images.githubusercontent.com/5543595/221561131-8b823d46-186d-4048-a35e-ea5072910ba0.png)

Packages:
```json
"stylelint": "^15.2.0",
"stylelint-config-property-sort-order-smacss": "^9.1.0",
"stylelint-config-standard": "^30.0.1",
"stylelint-order": "^6.0.2",
"stylelint-use-logical-spec": "^5.0.0"
```

Config:
```js
module. Exports = {
	plugins: ['stylelint-use-logical-spec', 'stylelint-order'],
	extends: ['stylelint-config-standard', 'stylelint-config-property-sort-order-smacss'],
	rules: {
		'alpha-value-notation': 'number',
		'block-no-empty': true,
		'color-function-notation': 'modern',
		'color-no-hex': true,
		'color-no-invalid-hex': true,
		'comment-empty-line-before': ['always', { except: ['first-nested'], ignore: ['stylelint-commands'] }],
		'comment-no-empty': true,
		'comment-whitespace-inside': 'always',
		'custom-property-empty-line-before': 'never',
		'custom-property-pattern': [
			'^([a-z][a-z0-9]*)(--?[a-z0-9]+)*$',
			{
				message: 'Expected custom property name to be kebab-case'
			}
		],
		'declaration-block-no-duplicate-properties': [true, { ignore: ['consecutive-duplicates-with-different-values'] }],
		'declaration-block-no-redundant-longhand-properties': true,
		'declaration-block-no-shorthand-property-overrides': true,
		'declaration-block-single-line-max-declarations': 1,
		'declaration-empty-line-before': 'never',
		'font-family-no-duplicate-names': true,
		'function-calc-no-unspaced-operator': true,
		'function-linear-gradient-no-nonstandard-direction': true,
		'function-name-case': 'lower',
		'hue-degree-notation': 'number',
		'keyframe-declaration-no-important': true,
		'length-zero-no-unit': true,
		'liberty/use-logical-spec': 'always',
		'media-feature-name-no-unknown': true,
		'no-empty-source': true,
		'no-descending-specificity': null /* TODO: enable this rule */,
		'no-invalid-double-slash-comments': true,
		'property-no-unknown': true,
		'rule-empty-line-before': [
			'always-multi-line',
			{ except: ['first-nested', 'inside-block-and-after-rule'], ignore: ['after-comment'] }
		],
		'selector-class-pattern': [
			'^([a-z][a-z0-9]*)(--?[a-z0-9]+)*$',
			{
				message: 'Expected class selector to be kebab-case'
			}
		],
		'selector-pseudo-class-no-unknown': true,
		'selector-pseudo-element-colon-notation': 'double',
		'selector-pseudo-element-no-unknown': true,
		'selector-type-case': 'lower',
		'selector-type-no-unknown': [true, { ignore: ['custom-elements'] }],
		'shorthand-property-no-redundant-values': true,
		'string-no-newline': true,
		'unit-no-unknown': true
	}
};

```

**Comment by Th3S4mur41** [comment]
The Code is reporting:

> logical shorthands do not work yet in browsers
> but we can still use shorthand if all values are the same

I could not find any documentation about it in the PR/Issue...
Is this referring to any of those tests?
https://wpt.fyi/results/css/css-logical/logical-box-padding.html?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&view=subtest&q=logical

**Comment by Th3S4mur41** [comment]
Just found that they are reported as experimental by [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Logical_Properties) in the context of logical properties.

**Comment by Th3S4mur41** [comment]
Thanks to [Romain Menke's explanation](https://github.com/csstools/postcss-plugins/issues/960#issuecomment-1534955878), the change now makes more sense and the reason for the suggestion is correct if you want to take advantage of logical properties
